spec/cocoapods/command/gen_spec.rb

```diff
@@ -47,6 +47,7 @@
 
               Options:
 
+                  --allow-root                           Allows CocoaPods to run as root
```

Comment:
Is this just an undocumented setting, or where is this setting coming from?

Reply:
its from 1.10.0.beta release with a new option to run CocoaPods as root. The bump in this PR caused this test to fail.